appcues/src/main/java/com/appcues/action/appcues/LaunchExperienceAction.kt

```diff
@@ -45,7 +50,9 @@ internal class LaunchExperienceAction(
 
     override suspend fun execute() {
         if (experienceId != null) {
-            experienceRenderer.show(experienceId, getTrigger())
+            coroutineScope.launch {
```

Comment:
here is the important change, basically launch a new scope before calling show(..)